Arize_Tutorials/Embeddings/GENERATIVE/Arize_Tutorial_LLM_Prompt_Engineering.ipynb

```diff
@@ -35,8 +35,7 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -q 'arize[AutoEmbeddings, LLM_Evaluation]' 
</pre><p>Can you remove the installation of extra deps? They are not needed as far as I can tell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/tutorials_python/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>